guide/getting-started/open-design.md

```diff
@@ -0,0 +1,112 @@
+- rely on our peer group for free and communal validation of quality
+- are accountable for our own actions, and have vested interest in the groups actions and goals
+
+And we value...
+
+- design ethics to make moral and responsible choices
+- human design to create products for real needs and desires
+- usability to create products that allow users to effectively solve their problems
+- accessibility to make products usable by people with the widest possible range of abilities, in the widest possible range of situations
+- aesthetics, which have a highly functional element and also influence how people feel and think
+
+## Getting started
+
+If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
+
+- Join the [Bitcoin Design Community](http://bitcoindesigners.org) on Slack and see what others are discussing. A good way to get started is by asking questions and joining conversations that interest you. Our [Community Calls](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) and [Design Review](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Design+Review+Call+) calls are also great ways to get active.
+- Download a Bitcoin wallet and experiment with small amounts. You can alse use Testnet, which allows you use all the features, but with worthless test bitcoin. Pay attention to the experience and come up with ways to improve it.
+- Redesign a Bitcoin application of your choice. Even as a brief exercise, a redesign is a great way to develop deeper understanding of how something works and why certain decisions were made.
+- Read, watch and listen. There is a lot of great material out there to learn more about Bitcoin. Design-focused content is unfortunately not as common but expanding into related materials (like finance) might help.
+- Review your tools and workflows, and the work you produce, and consider how you can make it more public and accesssible for others.
+
+## Licensing
+
+Licenses help ensure that bad actors cannot abuse this open system by taking the published work and pretend it's their own, patent it, redistribute it under a commercial license, or otherwise profit from it. A variety of licenses have been created by the open-source movement to accomodate different types of projects and situations.
+
+This guide is published under three different licenses ([MIT license](LICENSE-MIT), [Apache License 2.0](LICENSE-APACHE), and [Creative Commons Attribution 4.0 International License](LICENSE-CC-BY)). Anyone who wants to remix it, or parts of it, may choose which one to publish their work under. You can read the full license text [here](LICENSE).
```

Comment:
Remember to add the links in for these 👍

Reply:
Good call.